daemon/README.md

```diff
@@ -25,3 +25,16 @@ Currently the database for taker and maker is the same.
 The `taker` binary is used as an example to run the `prepare` command above, but it is irrelevant if you run it for taker or maker.
 The `tempdb` created can be deleted, it should not be checked into the repo.
 You can keep it around and just run the `prepare` statement multiple times when working on the database.
+
+## Logging
+
+`maker` and `taker` both make use of a log library which can be controlled using the environment variable `RUST_LOG`.
+By default, logging is set to `debug` for application specific logs only.
+If you want to see logs by third party libraries, you need to set `RUST_LOG=`.
+For example, to enable logs for rocket, use:
```

Comment:
I think we should retain rocket logs by default.

Reply:
Do you want that hardcoded `tracing.rs`?

Reply:
I would have just included `rocket=` the same way we have `taker=` and `maker=`.

Reply:
done